spec/factories/responses.rb

```diff
@@ -26,7 +26,10 @@
     type       { AnsibleTowerClient::FactoryHelper.underscore_string(klass.namespace.last) }
     name       { "#{type}-#{id}" }
     url        { "/api/v1/endpoint/" }
-    related    { {"survey_spec" => "example.com/api", 'inventory' => 'inventory link', 'stdout' => 'example.com/api'} }
+    related    { {"survey_spec" => "example.com/api",
```

Comment:
Avoid multi-line blocks with `{}`.

Reply:
To be honest, I do not like this using `{}` for one-liners and `do … end` for multi-line blocks much. They behave slightly differently (not in this case) and here it breaks the nice symmetry. I understand the style standards however, so fixed.